src/components/20-molecules/cookie-disclaimer/story.js

```diff
@@ -1,52 +1,25 @@
 /* global document */
 import { storiesOf } from '@storybook/html';
 import './index';
-import withNoBorder from '../../../../.storybook/addons/no-border';
 import Readme from './README.md';
+import withNoBorder from '../../../../.storybook/addons/no-border/no-border';
 
 storiesOf('Molecules/Cookie disclaimer', module)
-  .addDecorator(withNoBorder)
+  .addDecorator(story => {
+    return withNoBorder(story);
+  })
   .addParameters({
     readme: {
       sidebar: Readme,
     },
   })
-  .add('Cookie disclaimer - default', () => {
```

Comment:
All this mess code is here to show what happens when u click on it and shows u when u clicked as deno. Please leave this as is great demo value

Reply:
Maybe ask for the reason why someone does something before deleting. If u leave ur example, once someone clicked on accept he will always see blank when he opens storybook. Thats why i took the effort of making a nice example and not just because it looks bad means is bad. Is a demo anyway.